.travis.yml

```diff
@@ -14,6 +14,7 @@ matrix:
 env:
   global:
     - PYTEST_SETTINGS="not requires_gpu and not memory_intense and not slow and not travis_slow"
+    - CHANGES_PLUGINS="false"
```

Comment:
do we have to create this variable for it to be modified later on? Maybe we can add a comment explaining what this is for